### PR TITLE
Safegraph patterns docs: Additions and Revisions

### PR DESCRIPTION
**Pull request by Akvannortwick**
Based on suggestions made in #302.

Please add comments and suggestions on language as you see fit and I will make the adjustments.

### CODE REVIEW

**Comment by jingjtang** [comment]
I checked the geographic coverage at state level. Here is something that can be added to limitation or geographic exceptions.

```
The number of POIs coded as bars is much smaller than the number of POIs coded as restaurants. There is no bar visits data provided in SafeGraph's Weekly Patterns data for Alaska, Delaware, Maine, North Dakota, New Hampshire, South Dakota, Vermont, West Virginia, Wyoming consistently. For certain dates, bar visits data is also missing for District of Columbia, Idaho and Washington. Restaurant visits data is available for all of the states, District of Columbia and Puerto Rico.
```

**Comment by Akvannortwick** [comment]
If it should be in under "Geographic exception" let me know and I can make another section for that specifically.

File: docs/api/covidcast-signals/safegraph.md [hunk 1]
```diff
@@ -119,3 +119,11 @@ SafeGraph provides newly updated data for the previous week every Wednesday,
 meaning estimates for a specific day are only available 3-9 days later. It may
 take up to an additional day for SafeGraph's data to be ingested into the
 COVIDcast API.
+
+### Limitations
+
+This data source is based on provided through those that are part of SafeGraph's panel members. It is important to note that counts are subject to bias if some regions have more SafeGraph panel members. These counts do not represent absolute counts, only count in the panel in that region.
```
Comment (krivard) [inline docs/api/covidcast-signals/safegraph.md:125]:
```suggestion
This data source is based on mobile devices that are members of SafeGraph panels, which is not necessarily the same thing as measuring the general public. This means that counts are subject to bias if some regions have a greater density of SafeGraph panel members as a percentage of the population. These counts do not represent absolute counts, and only count visits by members of the panel in that region.
```

File: docs/api/covidcast-signals/safegraph.md [hunk 2]
```diff
@@ -119,3 +119,11 @@ SafeGraph provides newly updated data for the previous week every Wednesday,
...
+
+The number of POIs coded as bars is much smaller than the number of POIs coded as restaurants. 
+There is no bar visits data provided in SafeGraph's Weekly Patterns data for Alaska, Delaware, Maine, North Dakota, New Hampshire, South Dakota, Vermont, West Virginia, Wyoming consistently.
```
Comment (krivard) [inline docs/api/covidcast-signals/safegraph.md:128]:
```suggestion
SafeGraph's Weekly Patterns data consistently lacks data on bar visits for Alaska, Delaware, Maine, North Dakota, New Hampshire, South Dakota, Vermont, West Virginia, and Wyoming. 
```

"lacks" could also be "includes no"

File: docs/api/covidcast-signals/safegraph.md [hunk 3]
```diff
@@ -119,3 +119,11 @@ SafeGraph provides newly updated data for the previous week every Wednesday,
...
 take up to an additional day for SafeGraph's data to be ingested into the
 COVIDcast API.
+
+### Limitations
+
+This data source is based on provided through those that are part of SafeGraph's panel members. It is important to note that counts are subject to bias if some regions have more SafeGraph panel members. These counts do not represent absolute counts, only count in the panel in that region.
+
+The number of POIs coded as bars is much smaller than the number of POIs coded as restaurants. 
+There is no bar visits data provided in SafeGraph's Weekly Patterns data for Alaska, Delaware, Maine, North Dakota, New Hampshire, South Dakota, Vermont, West Virginia, Wyoming consistently. 
+For certain dates, bar visits data is also missing for District of Columbia, Idaho and Washington. Restaurant visits data is available for all of the states, District of Columbia and Puerto Rico.
```
Comment (krivard) [inline docs/api/covidcast-signals/safegraph.md:129]:
```suggestion
For certain dates, bar visits data is also missing for District of Columbia, Idaho and Washington. Restaurant visits data is available for all of the states, as well as the District of Columbia and Puerto Rico.
```

**Review by krivard — CHANGES_REQUESTED** [review]

**Review by krivard — APPROVED** [review]
👍

**Review by capnrefsmmat — APPROVED** [review]

**Pull request merged**